src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -103,20 +103,20 @@ <h2 class="b-rt-0 b-m-0 b-bold">Tip The Big Give</h2>
           </mat-hint>
         </div>
         <app-donation-tipping-slider
-            *ngIf="!showCustomTipInput"
+            [style]="showCustomTipInput ? 'display: none;' : ''"
             #donationTippingSlider
-            [percentageStart]="1"
+            [percentageStart]="minimumTipPercentage"
             [percentageEnd]="maximumTipPercentage"
             [donationAmount]="donationAmount"
             [donationCurrency]="campaign.currencyCode"
             [onHandleMoved]="onDonationSliderMove"
           />
 
-          <span class="button" (click)="toggleTipInput()" *ngIf="!showCustomTipInput && donationAmount">
+          <span class="button" (click)="displayCustomTipInput()" *ngIf="!showCustomTipInput && donationAmount">
             Other amount
           </span>
           <!-- TODO: this check sets an infinite loop, need to check the logic -->
-          <span *ngIf="shouldShowBackToSlider()"  class="button" (click)="toggleTipInput()" >
+          <span *ngIf="showCustomTipInput && tipIsWithinSuggestedPercentRange"  class="button" (click)="displayPercentageTipInput()" >
```

Comment:
using an expression rather than a function so ZoneJS can detect when to re-evaluate the expression automatically - afaik it won't re-run an explicit function call.